lib/prometheus_exporter/client.rb

```diff
@@ -9,18 +9,20 @@ class Client
     class RemoteMetric
       attr_reader :name, :type, :help
 
-      def initialize(name:, help:, type:, client:)
+      def initialize(name:, help:, type:, client:, opts: {})
```

Comment:
can't we just flow `opts: nil` all the way through and simply not send it over if it is nil?